docker/CENTOS_latest_NGEN_RUN.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM centos:8 as builder
```

Comment:
This dockerfile is named `CENTOS_latest` but builds from centos8 whereas `CENTOS_NGEN_RUN` actually pulls `centos:latest`

Reply:
Weirdly enough the :latest and :8 both pull the latest Centos8, but the :8 won't upgrade when the inevitable Centos 9 is out. I maybe need to label the 8_Latest better.